pseudocode.txt

```diff
@@ -35,11 +38,13 @@ VCF File with set of unique variants:
 
     Loop through rows in product of above tissue type subset
 	If genotype information is not blank
-	        If chrom, pos, ref, and alt (combination of them all) are unique
+	        If chrom, pos, ref, and alt (combination of them all) have not been added to the final vcf file
```

Comment:
This will help you make sure you got all the variants you wanted.  You might also think about how you could test that you didn't get any variants you **don't** want. One way to do this would be to collect variant annotations, such as from OpenCravat, and make sure you don't have any annotations for unexpected genes.